lib/transformations/bannerPlugin/bannerPlugin.js

```diff
@@ -1,9 +1,11 @@
+// @flow
+import type { Ijscodeshit, INewExpression, IPath } from '../../types'; // eslint-disable-line node/no-unsupported-features
```

Comment:
Prettify

Reply:
I've got some problem with the rebasing from `upstream` of my fork, I am in the process of fixing it. After that I will apply prettier to all my changes

Reply:
Prettier should have been applied during the precommit. It didn't work or did you skip it?

Reply:
It seems you don't have the latest changes that add prettier in this branch yet. Any ways you don't have to run prettier manually, just commit your files and they will be prettified. See a gif on https://github.com/okonet/lint-staged

Reply:
Yeah exactly that's what I was referring to. Somehow  the rebase didn't work as expected to I am in the processes to fix my previous commits manually so I will have prettier and eslint properly installed and working